JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -65,82 +71,75 @@ import com.example.jetnews.model.Paragraph
                 end = defaultSpacerSize,
                 bottom = 0.dp
             )
         ) {
-            Spacer(LayoutHeight(defaultSpacerSize))
+            Spacer(Modifier.preferredHeight(defaultSpacerSize))
             PostHeaderImage(post)
             Text(text = post.title, style = typography.h4)
-            Spacer(LayoutHeight(8.dp))
+            Spacer(Modifier.preferredHeight(8.dp))
             post.subtitle?.let { subtitle ->
-                ProvideEmphasis(emphasis = EmphasisLevels().medium) {
+                ProvideEmphasis(EmphasisAmbient.current.medium) {
                     Text(
                         text = subtitle,
                         style = typography.body2.merge(TextStyle(lineHeight = 20.sp))
                     )
                 }
-                Spacer(LayoutHeight(defaultSpacerSize))
+                Spacer(Modifier.preferredHeight(defaultSpacerSize))
             }
-            PostMetadata(metadata = post.metadata)
-            Spacer(LayoutHeight(24.dp))
-            PostContents(paragraphs = post.paragraphs)
-            Spacer(LayoutHeight(48.dp))
+            PostMetadata(post.metadata)
+            Spacer(Modifier.preferredHeight(24.dp))
+            PostContents(post.paragraphs)
+            Spacer(Modifier.preferredHeight(48.dp))
         }
     }
 }
 
 @Composable
 private fun PostHeaderImage(post: Post) {
     post.image?.let { image ->
-        // FIXME (dev07): This duplicate sizing is a workaround for the way Draw modifiers work in
-        // dev06. In a future release (dev07?) replace this with a single Image composable with a
-        // Clip modifier applied.
-
-        // This sizing is currently applied twice to make the image expand propertly inside the
-        // Clip.
-        val sizingModifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill
-        Box(modifier = sizingModifier) {
-            Clip(shape = RoundedCornerShape(4.dp)) {
-                val imageModifier = ImagePainter(image).toModifier(scaleFit = ScaleFit.FillHeight)
-                Box(modifier = sizingModifier + imageModifier)
-            }
-        }
-        Spacer(LayoutHeight(defaultSpacerSize))
+        val imageModifier = Modifier
+            .preferredHeightIn(minHeight = 180.dp)
+            .fillMaxWidth()
+            .clip(shape = RoundedCornerShape(4.dp))
+        Image(image, imageModifier, scaleFit = ScaleFit.FillHeight)
+        Spacer(Modifier.preferredHeight(defaultSpacerSize))
     }
 }
 
 @Composable
 private fun PostMetadata(metadata: Metadata) {
-    val typography = MaterialTheme.typography()
+    val typography = MaterialTheme.typography
     Row {
-        VectorImage(
-            id = R.drawable.ic_account_circle_black,
-            tint = contentColor()
+        Image(
+            asset = vectorResource(R.drawable.ic_account_circle_black),
+            colorFilter = ColorFilter(
+                color = contentColor(),
+                blendMode = BlendMode.srcATop)
```

Comment:
```suggestion
                blendMode = BlendMode.srcIn)
```

Reply:
Replaced with Icon, which does the right thing :)